.github/workflows/lint.yml

```diff
@@ -0,0 +1,12 @@
+name: Lint
+
+on: [push]
```

Comment:
I want the tests to run on 'pull-request' instead of 'push'

Reply:
I will fix it